lib/file-operations.js

```diff
@@ -363,6 +391,7 @@ WriteStream.prototype.open = function() {
 // Use our `end` method since it is patched for flush
 WriteStream.prototype.destroySoon = WriteStream.prototype.end;
 // Use node's `fs.WriteStream` methods
+WriteStream.prototype._destroy = fs.WriteStream.prototype._destroy;
```

Comment:
@erikkemperman this is why your latest tests were failing for node 8 - they changed `.destroy` to `._destroy` on the prototype.